src/azul/chalice.py

```diff
@@ -145,3 +153,107 @@ def _log_response(self, response):
     # Some type annotations to help with auto-complete
     lambda_context: LambdaContext
     current_request: Request
+
+
+class ValidatingAzulChaliceApp(AzulChaliceApp):
+    """
+    This class allows for a validation method to decorate the called view
+    function.
+    """
+
+    def __init__(self, app_name, unit_test=False, spec=None):
+        super().__init__(app_name=app_name, unit_test=unit_test, spec=spec)
+
+    # override so we can call _validate
+    def route(self, *args, validate: bool = True, **kwargs):
+        """
+        See docstring from super class for additional information.
+        :param validate: If False, do not validate requests submitted to the
+        endpoint.
+        """
+
+        azul_chalice_decorator = super().route(*args, **kwargs)
```

Comment:
The smaller the [scope](https://en.wikipedia.org/wiki/Scope_(computer_science)) the shorter the variables can be.

```suggestion
        super_decorator = super().route(*args, **kwargs)
```